vere/main.c

```diff
@@ -81,7 +81,7 @@ _main_getopt(c3_i argc, c3_c** argv)
   u3_Host.ops_u.fog = c3n;
   u3_Host.ops_u.gab = c3n;
   u3_Host.ops_u.git = c3n;
-  u3_Host.ops_u.has = c3n;
+  u3_Host.ops_u.has = c3y;
```

Comment:
In #1042, battery hashing is disabled for fake-keyed ships. Is that sufficient, or do you want it to be opt-in everywhere?